ports/nrf/modules/machine/uart.c

```diff
@@ -182,8 +202,10 @@ STATIC mp_obj_t machine_hard_uart_make_new(const mp_obj_type_t *type, size_t n_a
     config.p_context = (void *)self;
 
     // Set NULL as callback function to keep it blocking
```

Comment:
I guess we can delete this comment as it became a bit misleading :)

Reply:
Yes!